src/Data/Language.hs

```diff
@@ -93,7 +93,15 @@ extensionsForLanguage language = T.unpack <$> maybe mempty Lingo.languageExtensi
 
 -- | Return a language based on a FilePath's extension.
 languageForFilePath :: FilePath -> Language
-languageForFilePath path = maybe Unknown (textToLanguage . Lingo.languageName) (Lingo.languageForPath path)
+languageForFilePath path =
+  let spurious lang = lang `elem` [ "Hack" -- .php files
+                                  , "GCC Machine Description" -- .md files
+                                  , "XML" -- .tsx files
```

Comment:
Not sure why lingo thinks .tsx files could be XML, but a workaround was straightforward (and I added this to the tests).

Reply:
[This is why](https://github.com/tclem/lingo-haskell/blob/master/languages.yml#L5593).

Reply:
wow wtf

Reply:
something something XML and force something